docs/grpc/docs.md

```diff
@@ -3203,6 +3220,7 @@ For example, if `oversampling` is 2.4 and `limit` is 100, then 240 vectors will
 | version | [uint64](#uint64) |  | Last update operation applied to this point |
 | vectors | [Vectors](#qdrant-Vectors) | optional | Vectors to search |
 | shard_key | [ShardKey](#qdrant-ShardKey) | optional | Shard key |
+| order_value | [OrderedValue](#qdrant-OrderedValue) | optional | Order by value |
```

Comment:
Can we make the field name and the field type consistent?
Like order vs ordered?

Reply:
I have slight preference towards `order_value`

Reply:
I don't have a strong opinion to keeping different names, but it made sense for me to name it after OrderedFloat

Reply:
I think it is a bit different semantics here. We are not making Value ordered, we are using Value as order